lib/registration.pm

```diff
@@ -231,15 +231,15 @@ sub select_addons_in_textmode {
     if ($flag) {
         send_key_until_needlematch 'scc-module-area-selected', 'tab';
         send_key_until_needlematch "scc-module-$addon",        'down';
-        send_key 'spc';
+        wait_screen_change { send_key 'spc' };
```

Comment:
yes, I have seen this link but still "wait_screen_change+wait_still_screen". why should it make sense?